src/components/Modal/README.md

```diff
@@ -26,7 +26,9 @@ React.createElement(() => {
   const modalPrimaryButtonRef = React.useRef();
   const modalCloseButtonRef = React.useRef();
   return (
-    <>
+    <RUIProvider globalProps={{
+      Modal: { preventScrollUnderneath: window.document.documentElement }
+    }}>
```

Comment:
Can we have a comment explaining the reason in all examples? Example code snippets should be copy-paste ready (that's why I'd like to get rid of this workaround in the future, even if we have to modify the docs theme).

Reply:
> that's why I'd like to get rid of this workaround in the future, even if we have to modify the docs theme

That somehow implies that the docs theme is wrong. Is it wrong? If not then I see two options:

1.  Make the `<html>` element default. However changing defaults based on the docs platform is  is strange…
2.  Or we could make default the `off` value. That would be agnostic to the context of usage. I actaally kind of like this, because it makes no assumptions and it is expected that RUI should be tweaked before usage anyway.

Reply:
> That somehow implies that the docs theme is wrong. Is it wrong?

No. We just want to show examples **as simple as possible**, but with the current docs theme we are forced to showcase the more complicated version. And I still believe we chose the right default for `Modal`, i.e. to expect scrolling on `<body>`.

> it is expected that RUI should be tweaked before usage anyway

Oh, it doesn't need to be! 🙂 But that's another discussion…

According to our call today, I'm adding dropping a decent comment on it so we can move on.